doc/fitting.rst

```diff
@@ -493,6 +493,7 @@ place boundaries on this parameter one can do:
 Now we have to set up the minimizer and do the sampling:
 
 .. jupyter-execute::
+    :hide-output:
 
     res = lmfit.minimize(residual, method='emcee', nan_policy='omit', burn=300, steps=1000, thin=20,
```

Comment:
If you used here `steps=50000` you'd be able to compute the autocorrelation time. However, this takes about 10 minutes runtime.